### PR TITLE
OSS::RadioButton with documetation & tests 

### PR DESCRIPTION
**Pull request by Miexil**
### What does this PR do?
Adds a new version of the radiobuttons in the OSS section.

### Good PR checklist
- [x] Title makes sense
- [x] Is against the correct branch
- [x] Only addresses one issue
- [x] Properly assigned
- [x] Added/updated tests
- [x] Added/updated documentation
- [ ] Migrated touched components to Glimmer Components
- [x] Properly labeled

### CODE REVIEW

File: testem.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ module.exports = {
   browser_start_timeout: 120,
   browser_args: {
     Chrome: {
+      dev: ['--auto-open-devtools-for-tabs'],
```
Comment (Miexil) [inline testem.js:11]: FYI, this is the same as what I've added to publishr a while back.
It opens the dev-tools directly when running the test-server

**Review by JulienVannier66 — APPROVED** [review]
Otherwise LGTM

File: tests/integration/components/o-s-s/radio-button-test.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    assert.dom('.upf-radio-btn--disabled').exists();
+  });
+
+  test('It triggers the onChange method on click when the @disabled parameter is false', async function (assert) {
+    this.onChange = sinon.stub();
+    await render(hbs`<OSS::RadioButton @selected={{false}} @onChange={{this.onChange}} />`);
+    assert.dom('.upf-radio-btn').exists();
+    await click('.upf-radio-btn');
+    assert.true(this.onChange.calledOnceWithExactly(true));
+  });
```
[the remark this answers is not in the document]
Comment (Miexil) [reply tests/integration/components/o-s-s/radio-button-test.ts:47]: Sure !

**Review by JulienVannier66 — APPROVED** [review]

File: addon/components/o-s-s/radio-button.hbs [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<button class="upf-radio-btn {{if @selected 'upf-radio-btn--selected'}} {{if @disabled 'upf-radio-btn--disabled'}}"
```
Comment (phndiaye) [inline addon/components/o-s-s/radio-button.hbs:1]: pretty sure we have `upf-radio-btn` being used in the software already so just be careful with that one 😅

File: addon/components/o-s-s/radio-button.hbs [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+        {{on "click" this.onSelection}}
+        type="button" />
```
Comment (phndiaye) [inline addon/components/o-s-s/radio-button.hbs:3]: maybe add a `...attributes` so we can add a `data-control-name` if needed too

**Pull request merged**